src/main/java/yablunovskyi/bookstore/dto/BookDto.java

```diff
@@ -0,0 +1,15 @@
+package yablunovskyi.bookstore.dto;
+
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Data
+public class BookDto {
```

Comment:
it is better to use records for DTO (apply  to all dtos) 

upgrade your maven-checkstyle-plugin to 3.3.0 if Checkstyle fires at records